pkg/kv/kvserver/mvcc_gc_queue.go

```diff
@@ -58,8 +58,9 @@ const (
 
 	// Thresholds used to decide whether to queue for MVCC GC based on keys and
 	// intents.
-	mvccGCKeyScoreThreshold    = 2
-	mvccGCIntentScoreThreshold = 1
+	mvccGCKeyScoreThreshold          = 2
+	mvccGCIntentScoreThreshold       = 1
+	mvccGCDropRangeKeyScoreThreshold = 1.1
```

Comment:
Is there a reason for the 1.1 here? Seems like the FuzzFactor will only ever decrease the score, so there's no risk of it triggering a premature run. Might still be prudent though, to avoid premature runs for whatever other reason.

Reply:
I'm being paranoid in case fuzz factor is in the range of 0.95 to 1 and if that could trigger GC when range tombstones are just above threshold we'll get into worst possible collection case.

Reply:
Isn't it the opposite? If the fuzz factor is 0.95, it will wait 5% longer than necessary before triggering?

Reply:
Yep, but if we got right side of interval we can get to 1 and fire right at the moment? I guess it is very unlikely. Let me set that to 1.